rest-api-spec/src/main/resources/rest-api-spec/test/indices.get_mapping/50_wildcard_expansion.yml

```diff
@@ -2,40 +2,52 @@
 setup:
   - do:
         indices.create:
+          include_type_name: false
           index: test-xxx
           body:
             settings:
               index:
                 number_of_replicas: 0
             mappings:
-              type_1: {}
+              properties:
+                foo:
+                  type: keyword
   - do:
         indices.create:
+          include_type_name: false
           index: test-xxy
           body:
             settings:
               index:
                 number_of_replicas: 0
             mappings:
-              type_2: {}
+              properties:
+                foo2:
+                  type: keyword
```

Comment:
The test failed if I added an empty mappings object here. I think it might be because in the YAML framework we might evaluate {} as falsey but I could be wrong. If you feel strongly against doing this I could probably modify the checks below to check explicitly for an empty mappings object